server/service/base_client_test.go

```diff
@@ -66,22 +67,22 @@ func TestParseResponseOK(t *testing.T) {
 
 func TestParseResponseGeneralErrors(t *testing.T) {
 	t.Run("general HTTP errors", func(t *testing.T) {
-		bc, err := newBaseClient("https://test.com", true, "", "")
+		bc, err := newBaseClient("https://test.com", true, "", "", []fleet.Capability{})
 		require.NoError(t, err)
 		response := &http.Response{
 			StatusCode: http.StatusBadRequest,
-			Body:       ioutil.NopCloser(bytes.NewBufferString(`{"test": "ok"}`)),
+			Body:       io.NopCloser(bytes.NewBufferString(`{"test": "ok"}`)),
```

Comment:
Nice catch, maybe we should create a clean up issue to change all instances of `ioutil` to `io`.

Reply:
sounds good to me, so far I have been doing progressively each time I found an `ioutil` reference, but it might be worth it to do everything in a single pass to reduce noise in code reviews